client/assets/stylesheets/_wp-components-overrides.scss

```diff
@@ -158,12 +159,17 @@
 			}
 		}
 
+		&.is-secondary:not(:focus) {
+			box-shadow: inset 0 0 0 2px var(--wp-components-color-accent), 0 0 0 currentColor;
```

Comment:
Is it necessary to tweak the whole `box-shadow` style? Could we get by by only tweaking the shadow color?

Reply:
Woo has heavy overrides and uses a 2px thick line, so I think the whole thing is necessary

![calypso localhost_3000_log-in_redirect_to=https%3A%2F%2Fpublic-api wordpress com%2Foauth2%2Fauthorize%2F%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3D1c22dc5ce0975240c46902a714d400aa844e01ee0e530cfbfd8a6890603fa7cc%26redirect_ur (4)](https://github.com/user-attachments/assets/a4c46674-aa2c-45e5-94de-9301c6fbb7ed)